pkgs/applications/misc/swaymux/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, nlohmann_json
+, cmake
+, qt6
+, qtbase
+, qtwayland
+, stdenv
+, wrapQtAppsHook
+, fetchFromGitea
+}:
+stdenv.mkDerivation (finalAttrs: {
+  version = "1.0";
+  pname = "swaymux";
+
+  src = fetchFromGitea {
+    rev = "v${finalAttrs.version}";
+    domain = "git.grimmauld.de";
+    owner = "Grimmauld";
+    repo = "swaymux";
+    hash = "sha256-M85pqfYnYeVPTZXKtjg/ks5LUl3u2onG9Nfn8Xs+BSA=";
+  };
+
+  buildInputs = [ qt6.qtwayland cmake nlohmann_json qtbase];
+  nativeBuildInputs = [ cmake qt6.wrapQtAppsHook ];
+
+  doCheck = true;
+
+  meta = with lib; {
```

Comment:
Missing `meta.mainProgram`.

Reply:
Alright, i added `meta.mainProgram` as well as put parameters and meta entries into alphabetical ordering as suggested.